### PR TITLE
Add update step for change tags for aws_backup_vault resource

### LINKED ISSUE #7801: aws_backup_vault forces new resource when updating tags
### Terraform Version

Terraform 0.11.11
AWS 1.60

### Affected Resource(s)

aws_backup_vault

### Terraform Configuration Files

```hcl
variable "tags { 
    default = {
      environment = "test
   }
}

resource "aws_backup_vault" "backup_vault" {
  name        = "${var.name}"
  kms_key_arn = "${local.kms_arn}"
  tags        = "${merge(var.tags,map("name",var.name))}"
}

```

### Expected Behavior

Altering the tags for the resource like other resources

### Actual Behavior

(forces new resource) and tries to delete the resource which is blocked by existing backup resource
### Steps to Reproduce

1. Create aws_backup_vault resource 
2. `terraform apply`
3. Add Tag 
4. `terraform apply`

### References

* #7166

### PR DESCRIPTION
**Pull request by kterada0509**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

Fixes #7801 

Changes proposed in this pull request:

* Add step for update tags

Output from acceptance testing:

```
$ make testacc TEST=./aws TESTARGS='-run=TestAccAwsBackupVault_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -parallel 20 -run=TestAccAwsBackupVault_ -timeout 120m
=== RUN   TestAccAwsBackupVault_basic
=== PAUSE TestAccAwsBackupVault_basic
=== RUN   TestAccAwsBackupVault_withKmsKey
=== PAUSE TestAccAwsBackupVault_withKmsKey
=== RUN   TestAccAwsBackupVault_withTags
=== PAUSE TestAccAwsBackupVault_withTags
=== CONT  TestAccAwsBackupVault_basic
=== CONT  TestAccAwsBackupVault_withTags
=== CONT  TestAccAwsBackupVault_withKmsKey
--- PASS: TestAccAwsBackupVault_basic (31.67s)
--- PASS: TestAccAwsBackupVault_withTags (52.90s)
--- PASS: TestAccAwsBackupVault_withKmsKey (65.63s)
PASS
ok  	github.com/terraform-providers/tmp-terraform-provider-aws/aws	65.699s
```

### CODE REVIEW

**Comment by slapula** [comment]
LGTM

File: aws/resource_aws_backup_vault_test.go [hunk 1]
```diff
@@ -68,6 +68,15 @@ func TestAccAwsBackupVault_withTags(t *testing.T) {
...
 			},
+			{
+				Config: testAccBackupVaultWithUpdateTags(rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsBackupVaultExists("aws_backup_vault.test", &vault),
+					resource.TestCheckResourceAttr("aws_backup_vault.test", "tags.%", "2"),
+					resource.TestCheckResourceAttr("aws_backup_vault.test", "tags.foo", "bar"),
+					resource.TestCheckResourceAttr("aws_backup_vault.test", "tags.fizz", "buzz"),
+				),
+			},
```
Comment (slapula) [inline aws/resource_aws_backup_vault_test.go]: You may want to test adding/removing tags too

Comment (kterada0509) [reply aws/resource_aws_backup_vault_test.go]: Thanks. Added test case.

```
$ make testacc TEST=./aws TESTARGS='-run=TestAccAwsBackupVault_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -parallel 20 -run=TestAccAwsBackupVault_ -timeout 120m
=== RUN   TestAccAwsBackupVault_basic
=== PAUSE TestAccAwsBackupVault_basic
=== RUN   TestAccAwsBackupVault_withKmsKey
=== PAUSE TestAccAwsBackupVault_withKmsKey
=== RUN   TestAccAwsBackupVault_withTags
=== PAUSE TestAccAwsBackupVault_withTags
=== CONT  TestAccAwsBackupVault_basic
=== CONT  TestAccAwsBackupVault_withTags
=== CONT  TestAccAwsBackupVault_withKmsKey
--- PASS: TestAccAwsBackupVault_basic (33.29s)
--- PASS: TestAccAwsBackupVault_withKmsKey (66.19s)
--- PASS: TestAccAwsBackupVault_withTags (74.41s)
PASS
ok  	github.com/terraform-providers/tmp-terraform-provider-aws/aws	74.504s
```

**Comment by bflad** [comment]
This has been released in [version 2.2.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#220-march-15-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Pull request merged**